specification/hugr.md

```diff
@@ -2011,7 +2015,8 @@ including `Module`.
 | `AliasDefn`    | 0, 0    | 0, 0    | 0, 0    | 0, 0       | 0, 0          | 1, 0        |          |
 | `AliasDecl`    | 0, 0    | 0, 0    | 0, 0    | 0, 0       | 0, 0          | 1, 0        |          |
 | `Const`        | 0, 0    | 0, 0    | 0, ✱    | 0, 0       | 0, 0          | 1, 0        |          |
-| `LoadConstant` | 0, 1    | +, ✱    | 1, 0    | 0, 0       | 0, 0          | 1, 0        |          |
+| `LoadConstant` | 0, 1    | +, ✱    | 0, 0    | 1, 0       | 0, 0          | 1, 0        |          |
+| `LoadFunction` | 0, 1    | +, ✱    | 1, 0    | 0, 0       | 0, 0          | 1, 0        |          |
```

Comment:
Unrelated to this PR, but I think the `+` indicating "at least one incoming Order edge" for `LoadConstant` and `LoadFunction` may be outdated and should be `✱`.